### ISSUE TITLE
Button to quickly enable/disable voice messages?

### ISSUE THREAD

**Issue by rafnov** [organicmaps/organicmaps#9468]
For now you have to enter settings to do that. Making button to quickly enable/disable voice guidance would be much appreciated. 
In some cases OM is unusable because of annoying, useless voice messages on my simple routes, where it is quite obvious where to drive. Those spam messages make my music mute to often, but I would need them in some other cases, where route is not so obvious.

**Comment by Rob-from-VI** [comment]
I think it should take the form of a "mute" button,  showing crossed out after being pressed.

Very handy indeed.   
![image](https://github.com/user-attachments/assets/385008e5-fb90-4346-8c23-5f40c0b84210)

**Comment by rafnov** [comment]
Until I posted opinion in Google Play I did not know that pressing bottom bar in navigation mode shows additional options,  button to disable voice amongst them.

**Comment by patepelo** [comment]
Yes, the option is there. The question is, how should the UI improve to make it more clear?

**Comment by rafnov** [comment]
Showing it on main screen would be very... usable.

**Comment by Rob-from-VI** [comment]
I should have been clear, when talking about "while Navigating" I was thinking  about android auto.   I do not rercall seeing such a button on my car screen, which is where I would like to see it!!!

**Comment by rafnov** [comment]
@patepelo, I was not writing my post about Android Auto, but regular Organic Maps navigation.

**Comment by Rob-from-VI** [comment]
In that case for use on the phone itself, I personally feel the present implementation is perfectly good enough, as it avoids cluttering the screen.

Just touch the bottom bar and hit the mute button.

**Comment by rafnov** [comment]
One may that need, second another need, so why not make that configurable via toggle: show/dont show mute button on main screen?

**Comment by bbqman7089** [comment]
The other two big mapping softwares (Apple and Google) have this as a feature I believe. 

Even though it is just a single tap more, I personally prefer to have my eyes on the road instead of a small phone screen tapping away to shut the voice instructions up when they are not needed. Having a decently sized (doesn't have to be massive) button that can do this whilst navigating would vastly improve my UX.

**Comment by daboudii** [comment]
The option is way to hidden, if I need to look in a Github issue to find it there is a problem. 

Maybe while the start button is display the mute and settings should be display under and after pressing start with the animation it close. So when you lunch you have a visual cue that there more under the bar.  So no need for implementing.
